server/package.json

```diff
@@ -16,7 +17,8 @@
     "express": "^4.18.2",
     "jsonwebtoken": "^9.0.0",
     "mongoose": "^6.7.3",
-    "node-fetch": "^3.3.0"
+    "node-fetch": "^3.3.0",
+    "nodemon": "^2.0.21"
```

Comment:
i know this is handy tool, but can we wait to install it for now until it's decided? you can install nodemon globally on your computer for now if you want to use it.

Reply:
removed nodemon